app/controllers/katello/api/v2/content_view_versions_controller.rb

```diff
@@ -82,8 +82,22 @@ def republish_repositories
       respond_for_async :resource => task
     end
 
+    api :GET, "/content_view_versions/export_histories", N_("Show the export history for a content view version")
+    param :content_view_version_id, :number, :desc => N_("Content view version identifier"), :required => false
+    param :destination_server, String, :desc => N_("Destination Server name")
+    param :organization_id, :number, :desc => N_("Organization identifier")
+    def export_histories
+      history = ContentViewVersionExportHistory.readable
+      history = history.where(:content_view_version_id => params[:content_view_version_id]) unless params[:content_view_version_id].blank?
```

Comment:
would filtering by content view be more useful than by version?  or at least having it as an option too?

Reply:
Add content view version also.